app/controllers/vm_controller.rb

```diff
@@ -11,7 +11,9 @@ def index
   end
 
   def show_list
-    process_show_list(:association => session[:vm_type])
+    options = {:association => session[:vm_type]}
+    options[:model] = "ManageIQ::Providers::CloudManager::Vm" if params['sb_controller'] == 'availability_zone'
```

Comment:
@h-kataria As far as I can tell, the other cases (for example when viewing the vms and instances from workloads should be working fine).